src/api/routers/mcp.py

```
@@ -0,0 +1,389 @@
+
+class ToolExecuteResponse(BaseModel):
+    """Response from tool execution."""
+
+    tool: str
+    success: bool
+    content: List[Dict[str, Any]] = Field(default_factory=list)
+    error: Optional[str] = None
+    execution_time: float = 0.0
+
+
+def _get_mcp_manager(runtime: AutonomousRuntime) -> Optional[Any]:
```

Ruff (0.14.10)

Unused function argument: `runtime`

(ARG001)